architecture/3-DataStructures/6-InterfaceListItem.yaml

```diff
@@ -0,0 +1,23 @@
+name: Component List Item
```

Comment:
`Component List Item` should say `Interface List Item`

Reply:
Resolved [4cbcc20](https://github.com/ODU-CGA-CubeSat/dof-cubesat/pull/2/commits/4cbcc2047b672037edb01d55d017f9938b041063)